src/common/seed/images.ts

```diff
@@ -105,14 +113,21 @@ const generateDummyImage = () => ({
 	started_at: new Date(),
 	targetdevice: generateTargetDevice(),
 	thumb: generateRandomImage(),
-	build: generateBuild(),
+	build: generateBuild(forStatic, counter),
 });
 
 export const generateDummyImages = (length: number) => {
-	const images = Array.from(
-		{length},
+	const staticImages = Array.from(
+		{length: 3},
+		(v, index) => generateDummyImage(true, index),
+	);
+
+	const restImages = Array.from(
+		{length: length - 3},
 		() => generateDummyImage(),
 	);
 
+	const images = staticImages.concat(restImages);
```

Comment:
We want to have 3 images generated with different statuses to be placed first

Reply:
That's exactly how they ```concat```s right now. ```staticImages``` Is that 3 images